### DISCUSSION TITLE
New translation tool

### DISCUSSION THREAD

**Discussion by oguzkaganeren** [oguzkaganeren/manjaro-starter#368] [Announcements]
Hi guys, I added translation files to the poeditor. It synced with GitHub repo. You can continue in which way that is easy for you. I have not used it before. Maybe need some configuration, but it looks okay to me for now.

https://poeditor.com/join/project/E3GYwQeAVc

**Comment by oguzkaganeren** [comment]
Also, we have been approved as an open source project. 
![image](https://user-images.githubusercontent.com/5963437/216932814-40f30ee6-2ca5-4441-afad-4c126ce0615a.png)

**Comment by dai5m5iq** [comment]
Will have to try I guess :). Thanks for forwarding us.
